sycl/include/sycl/platform.hpp

```diff
@@ -57,6 +62,45 @@ namespace ext::oneapi {
 class filter_selector;
 } // namespace ext::oneapi
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+template <typename ParamT, typename PropT>
```

Comment:
How is `PropT` used?

Reply:
Oh, I forgot. Thanks. I will fix it quickly.

Reply:
Done. Thanks.